apps/src/templates/rubrics/StudentSelector.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import i18n from '@cdo/locale';
+import {updateQueryParam} from '@cdo/apps/code-studio/utils';
+import {reload} from '@cdo/apps/utils';
+import {queryUserProgress} from '@cdo/apps/code-studio/progressRedux';
+
+const NO_SELECTED_SECTION_VALUE = '';
+
+const styles = {
+  select: {
+    width: 180,
+  },
+};
```

Comment:
I think we want to keep style information in the scss module and avoid styling in our jsx files.